src/pages/GrammarPracticePage.css

```
@@ -2998,6 +3002,30 @@
 }
 
+.grammar-practice-next-grammar-note-plus.is-saved {
+  background: var(--dojeon-color-primary-action);
+  color: var(--dojeon-color-surface);
+  cursor: default;
+}
+
+.grammar-practice-next-grammar-note-plus.is-saved::before {
+  width: 9px;
+  height: 5px;
+  border: solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)